salt/_states/containerd.py

```diff
@@ -114,5 +90,45 @@ def image_managed(name, archive_path=None):
             ret['result'] = True
         else:
             ret['comment'] = 'Failed to pull image'
+        return ret
+
+    return ret
+
 
+def manage_archive_path(name, archive_path, ret):
```

Comment:
This naming seems not in line with other states. `manage` is active (i.e. a module).

If the intent is for this to be not 'public API of the state module' make it `_manage_archive_path` or something, though even then, even utility procedures in a state should be... a state-like thing.

Reply:
Reverting this since I'm dropping Python linting.